### PR TITLE
Except Any types from casting, and add an enum example for PostgreSQL

### PR DESCRIPTION
**Pull request by kaare**
If a pg type isn't found, it becomes an Any type. Can't immediately typecast those.
Also, add a test for enum types in Pg.

### CODE REVIEW

File: lib/DBDish/Pg/StatementHandle.pm6 [hunk 1]
```diff
@@ -94,7 +94,7 @@ method _row() {
                 $value = $!result.PQgetvalue($!current_row, $col);
                 if ct ~~ Array {
                     $value = _pg-to-array($value, ct.of);
-                } else {
+    			} elsif (ct.^name ne 'Any') {
```
Comment (jonathanstowe) [inline lib/DBDish/Pg/StatementHandle.pm6]: Maybe "ct === Any" rather than the string comparison.

Also wonky indentation :)

File: t/36-pg-enum.t [hunk 2]
```diff
@@ -0,0 +1,99 @@
+use v6;
+use Test;
+use DBIish;
+
+plan 27;
+
+my %con-parms;
+# If env var set, no parameter needed.
+%con-parms<database> = 'dbdishtest' unless %*ENV<PGDATABASE>;
```
Comment (jonathanstowe) [inline t/36-pg-enum.t]: I'm not sure we do actually want to do this anymore  (see #77 for discussion,)

**Comment by kaare** [comment]
'===' doesn't work, as it's always true.

**Comment by jonathanstowe** [comment]
Right, well I'll merge this (it passes its tests,) and then have al ook at that as the name comparison isn't ideal

**Pull request merged**